sormas-backend/src/main/java/de/symeda/sormas/backend/systemevent/SystemEventFacadeEjb.java

```diff
@@ -150,7 +150,6 @@ public void deleteAllDeletableSystemEvents(int daysAfterSystemEventGetsDeleted)
 	}
 
 	public void deleteAllDeletableSystemEvents(LocalDateTime notChangedUntil) {
-
```

Comment:
This formatting violates [formatting guidelines](https://github.com/hzi-braunschweig/SORMAS-Project/blob/development/docs/CONTRIBUTING.md#development-contributing-guidelines)
> When defining a method, enter a blank line before starting to write its body (except for methods with only one line of code, e.g. most getters and setters).